x-pack/libbeat/management/config.go

```diff
@@ -11,8 +11,9 @@ import (
 
 // Config for central management
 type Config struct {
-	Enabled   bool                    `config:"enabled" yaml:"enabled"`
-	Blacklist ConfigBlacklistSettings `config:"blacklist" yaml:"blacklist"`
+	Enabled       bool                    `config:"enabled" yaml:"enabled"`
+	Blacklist     ConfigBlacklistSettings `config:"blacklist" yaml:"blacklist"`
+	OutputRestart bool                    `config:"output_restart" yaml:"output_restart"`
```

Comment:
nit: We used `restart_on_output_change` as the configuration and yaml name here in https://github.com/elastic/beats/pull/34049/files so that it would match exactly with the parameter name in the [v1 spec files](https://github.com/elastic/elastic-agent/blob/1fa24b7a28c0e9e30647f17d945a03168055bde5/internal/spec/filebeat.yml#L12).

I also think `restart_on_output_change` more obviously describes what this does if you read it by itself on the spec file side.

Reply:
I updated to use the same flag name `restart_on_output_change`.